src/routes/index.ts

```diff
@@ -23,6 +28,35 @@ export const loginRateLimiter = createRateLimitMiddleware({
 	lifetime: 2 * SECONDS_PER_HOUR, // reset counter after 2 hours (in seconds)
 });
 
+export const registerDeviceStateEvent = (pathToUuid: _.PropertyPath) => {
```

Comment:
```suggestion
export const registerDeviceStateEvent = (pathToUuid: _.PropertyPath): RequestHandler => {
```
and remove the `as RequestHandler`s within the function